test/cypress/e2e/account.spec.js

```diff
@@ -18,41 +18,39 @@ describe('Account', () => {
     // https://github.com/cypress-io/cypress/issues/695
     cy.wait(1000);
     cy.url().should('contain', '16313739661666666666L');
-    cy.get(ss.accountAddress).eq(1).contains('16313739661666666666L');
-    cy.get(ss.accountBalance).eq(1).contains('0');
+    cy.get(ss.accountAddress).contains('16313739661666666666L');
```

Comment:
in line 35 you are using constants but not in here, maybe you can use constants in here too